Gems/Maestro/Code/Source/Cinematics/AnimSequence.cpp

```diff
@@ -57,16 +57,21 @@ CAnimSequence::CAnimSequence(uint32 id, SequenceType sequenceType)
 
     m_pEventStrings = aznew CAnimStringTable;
     m_expanded = true;
+
+    AZ_Info("CAnimSequence", "CAnimSequence type %i", static_cast<int>(sequenceType));
 }
 
 //////////////////////////////////////////////////////////////////////////
 CAnimSequence::CAnimSequence()
     : CAnimSequence(0, SequenceType::SequenceComponent)
 {
+    AZ_Info("CAnimSequence", "CAnimSequence no args");
```

Comment:
maybe put this behind AZ_ENABLE_TRACING, as for the other logs that you are adding